openmc/deplete/nuclide.py

```diff
@@ -123,6 +123,11 @@ def __init__(self, name=None):
         # Neutron fission yields, if present
         self._yield_data = None
 
+    def __repr__(self):
+        return "<Nuclide: {} ({} modes, {} reactions)>".format(
+            self.name, self.n_decay_modes, self.n_reaction_paths
+        )
+
```

Comment:
```suggestion
    def __repr__(self):
        return f"<Nuclide: {self.name} ({self.n_decay_modes} modes, {self.n_reaction_paths} reactions)>"

```

Reply:
This is actually what I had originally written, but it makes the line stretch out to column 105, so we'd still need a way of splitting it up over multiple lines and the other options seemed less palatable than just falling back to a `.format(...)` call, which is what I ended up doing

Reply:
fair enough, I started trying to do brackets ```(...)``` for a multi-line string but am not sure it looks any better

Reply:
```.format``` is fine, here is just one more option for your consideration

```suggestion
    def __repr__(self):
        return (
            f"<Nuclide: {self.name} ({self.n_decay_modes} modes, "
            f"{self.n_reaction_paths} reactions)>"
        )

```

Reply:
I adopted a slightly different approach -- create two (short-named) local variables so that the f-string fits on a single line. I think the end result is the most readable. Hope it meets your f-string standards :smile: